dashboard/test/factories.rb

```diff
@@ -475,6 +475,7 @@
   end
   factory :plc_course, class: 'Plc::Course' do
     name "MyString"
+    id Plc::Course.maximum(:id).try(:next) || 1
```

Comment:
I'm not clear how this fixes anything. It seems like it's just overwriting the fixture by id with new values, and working around the problem. Why was the fixture a problem in the first place? Is some other part of the test doing something with the first `Plc::Course`?